### ISSUE TITLE
Missing Aria Label

### ISSUE THREAD

**Issue by brianteeman** [cmfcmf/docusaurus-search-local#133]
### Please describe the bug you are seeing

When the search box is not expanded the code says that the search box is labelled by a non existing label

**Title: WCAG 4.1.2: Ensures every ARIA input field has an accessible name (.aa-Autocomplete)**
_Tags: Accessibility, WCAG 4.1.2, aria-input-field-name_

Issue: Ensures every ARIA input field has an accessible name (aria-input-field-name - https://accessibilityinsights.io/info-examples/web/aria-input-field-name)

Element path: .aa-Autocomplete

Snippet: <div class="aa-Autocomplete" role="combobox" aria-expanded="false" aria-haspopup="listbox" aria-labelledby="autocomplete-0-label">

 aria-labelledby attribute does not exist

Environment: Chrome version 104.0.0.0

====

### How can we best reproduce the bug?

Use any accessibility testing tool 

### Which version of Docusaurus and @cmfcmf/docusaurus-search-local are you using?

PS C:\laragon\www\Manual> npm list '@docusaurus/core' '@cmfcmf/docusaurus-search-local'
manual@0.0.0 C:\laragon\www\Manual
├─┬ @cmfcmf/docusaurus-search-local@0.11.0
│ └── @docusaurus/core@2.0.1 deduped
├── @docusaurus/core@2.0.1
└─┬ @docusaurus/preset-classic@2.0.1
  ├── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-content-blog@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-content-docs@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-content-pages@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-debug@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-google-analytics@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-google-gtag@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/plugin-sitemap@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  ├─┬ @docusaurus/theme-classic@2.0.1
  │ └── @docusaurus/core@2.0.1 deduped
  └─┬ @docusaurus/theme-search-algolia@2.0.1
    └── @docusaurus/core@2.0.1 deduped

### Which Node.js version are you using?

v18.3.0

### Which theme are you using?

Yes, I am using @docusaurus/theme-classic

### How does your plugin config look like?

```
  plugins: [
    [
      require.resolve("@cmfcmf/docusaurus-search-local"),
      {
        indexBlog: false,
        language: "en",
      },
    ],
  ],
```
